### PR TITLE
Create BLorient5268 (CH_march24)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient5268 according to the description by S. Strelcyn.

I used LIT4711Mastabqwe for p3_i1 according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2500#issuecomment-1999319043. However, I wonder, if I should nevertheless mention the manuscript BnF éth. 309 as S. Strelcyn did in this place or if I should omit it.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient5268.xml [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+                     <msPart xml:id="p3">
+                         <msIdentifier></msIdentifier>
+                         <msContents><summary/>
+                            <msItem xml:id="p3_i1">
+                                <locus from="161" to="162"/>
+                                <title ref="LIT4711Mastabqwe" type="incomplete">Deprecatory prayers</title>
+                                <incipit xml:lang="gez"><title>መስተበቍዕ፡</title></incipit>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">36</citedRange>
+                                    </bibl> several deprecatory prayers (mastabqʷǝʿāt) similar to those attested in 
+                                    <ref type="mss" corresp="0000000000000000000000000000000">BnF, éth. 309</ref>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient5268.xml:308]: yes you can refer, the ID should follow the pattern for other Éthiopien mss, we have not gotten to the Griaule collection yet, but surely you can also provide the actual folio numbers `<ref type="mss" corresp="BNFet309">BnF, éth. 309, ff. 72-75</ref>`

Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient5268.xml:308]: OK. But would it not be better to create a stub for BnF, éth. 309? In this case, BNFet309 would actually exist at the time this record is merged.

File: LondonBritishLibrary/orient/BLorient5268.xml [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+                                    <measure unit="leaf">1+163</measure>
+                                    <measure unit="leaf" type="blank" corresp="#I">1</measure>
+                                </extent>
+                            </supportDesc>    
+                        </objectDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#Iv"/>
+                                    <desc type="Unclear">Palimpsest on the verso of the front flyleaf.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient5268.xml:52]: Is this really a palimpsest? The leaf is made of paper. MAybe it is an erased note?

File: LondonBritishLibrary/orient/BLorient5268.xml [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+                                </layoutDesc>
+                            </objectDesc>  
+                        </physDesc>
+                    </msPart>
+                     <msPart xml:id="p3">
+                         <msIdentifier></msIdentifier>
+                         <msContents><summary/>
+                            <msItem xml:id="p3_i1">
+                                <locus from="161" to="162"/>
+                                <title ref="LIT4711Mastabqwe" type="incomplete">Deprecatory prayers</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient5268.xml:304]: I am confused by "deprecatory". The prayers of LIT4711Mastabqwe, as I am "familiar" (not very) with them, are rather supplicative.

**Review by thea-m — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient5268.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient5268.xml:52]: You are right, this information given by Strelcyn is somehow contradictory and confusing. He says, that there is one unnumbered flyleaf, which is unnumbered and that the "paper flyleaves (!) were supplied, when the manuscript was repaired in the B.M.". If I think about it once again, I come to the conclusion, that there are one (parchment) flyleaf (with a palimpsest) + (at least two) paper flyleaves. I will update the record accordingly. I do not assume, that Strelcyn counted an erased portion of a text as a palimpsest.

File: LondonBritishLibrary/orient/BLorient5268.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient5268.xml:304]: I can delete the verbal title, because the LIT is sufficient.

**Comment by CarstenHoffmannMarburg** [comment]
@thea-m 
I think, I made the changes, that you requested. Can this branch be merged?

**Pull request merged**